pydantic_ai_slim/pydantic_ai/models/mistral.py

```diff
@@ -397,6 +397,9 @@ def _get_python_type(cls, value: dict[str, Any]) -> str:
         # 5) Object: Check for additionalProperties
         if value_type == 'object':
             additional_properties = value.get('additionalProperties', {})
+            if isinstance(additional_properties, bool):
```

Comment:
To have coverage, we need to upgrade to 2.11. Can I upgrade to the latest beta in this PR?